internal/usecase/interactor/user.go

```diff
@@ -237,6 +237,61 @@ func (i *User) GetUserBySubject(ctx context.Context, sub string) (u *user.User,
 	return u, nil
 }
 
+func (i *User) PasswordResetRequest(ctx context.Context, param interfaces.PasswordResetRequestParam) (bool, error) {
```

Comment:
```suggestion
func (i *User) StartPasswordReset(ctx context.Context, email string) error {
```

Only error is enough. If no error returns, this operation was succeeded. Also param struct does not need because there are few params.